scripts/install.sh

```diff
@@ -0,0 +1,263 @@
+        ;;
+    *)
+        echo "Unsupported architecture: $ARCH"
+        exit 1
+        ;;
+esac
+
+echo "Detected architecture: $ARCH_SUFFIX"
+echo "Installing AutoDBA under: $PARENT_DIR"
+
+# Function to install Prometheus in a custom path
+install_prometheus() {
+    if [ -f "$PROMETHEUS_INSTALL_DIR/prometheus" ]; then
+        echo "Prometheus is already installed."
+    else
+        echo "Installing Prometheus in custom path: $PROMETHEUS_INSTALL_DIR..."
+        wget -qO- https://github.com/prometheus/prometheus/releases/download/v${PROMETHEUS_VERSION}/prometheus-${PROMETHEUS_VERSION}.linux-${ARCH_SUFFIX}.tar.gz | tar -xzf - -C /tmp/
+        mkdir -p "$PROMETHEUS_INSTALL_DIR"
+        cp /tmp/prometheus-${PROMETHEUS_VERSION}.linux-${ARCH_SUFFIX}/prometheus "$PROMETHEUS_INSTALL_DIR/"
+        cp /tmp/prometheus-${PROMETHEUS_VERSION}.linux-${ARCH_SUFFIX}/promtool "$PROMETHEUS_INSTALL_DIR/"
+        mkdir -p ${PROMETHEUS_CONFIG_DIR}
+        cp -r /tmp/prometheus-${PROMETHEUS_VERSION}.linux-${ARCH_SUFFIX}/consoles ${PROMETHEUS_CONFIG_DIR}/
+        cp -r /tmp/prometheus-${PROMETHEUS_VERSION}.linux-${ARCH_SUFFIX}/console_libraries ${PROMETHEUS_CONFIG_DIR}/
+    fi
+}
+
+# Function to install from .tar.gz
+install_tar_gz() {
+    # Ensure required directories exist
+    echo "Creating directories..."
+    mkdir -p "${INSTALL_DIR}" "${WEBAPP_DIR}" "${EXPORTER_DIR}" "${PROMETHEUS_CONFIG_DIR}" "${AUTODBA_CONFIG_DIR}" "${PROMETHEUS_STORAGE_DIR}"
+    echo "Extracting .tar.gz package $PACKAGE_FILE..."
+    tar -xzvf "$PACKAGE_FILE" -C /tmp/
+    
+    cp /tmp/autodba-*/bin/autodba-bff-${ARCH_SUFFIX} "${INSTALL_DIR}/autodba-bff"
+    cp -r /tmp/autodba-*/webapp/* "${WEBAPP_DIR}/"
+
+    # Copy each exporter into its own directory
+    mkdir -p "${EXPORTER_DIR}/postgres_exporter"
+    mkdir -p "${EXPORTER_DIR}/sql_exporter"
+    mkdir -p "${EXPORTER_DIR}/rds_exporter"
+    
+    cp -r /tmp/autodba-*/exporters/${ARCH_SUFFIX}/postgres_exporter/* "${EXPORTER_DIR}/postgres_exporter/"
+    cp -r /tmp/autodba-*/exporters/${ARCH_SUFFIX}/sql_exporter/* "${EXPORTER_DIR}/sql_exporter/"
+    cp -r /tmp/autodba-*/exporters/${ARCH_SUFFIX}/rds_exporter/* "${EXPORTER_DIR}/rds_exporter/"
+
+    cp /tmp/autodba-*/monitor/prometheus/prometheus.yml "${PROMETHEUS_CONFIG_DIR}/prometheus.yml"
+    cp -r /tmp/autodba-*/monitor/prometheus/sql_exporter/* "${EXPORTER_DIR}/sql_exporter/"
+    cp -r /tmp/autodba-*/monitor/prometheus/rds_exporter/* "${EXPORTER_DIR}/rds_exporter/"
+    cp /tmp/autodba-*/config/config.json "${AUTODBA_CONFIG_DIR}/config.json"
+    cp /tmp/autodba-*/entrypoint.sh "${INSTALL_DIR}/autodba-entrypoint.sh"
+
+    chmod +x "${INSTALL_DIR}/autodba-entrypoint.sh"
+}
+
+# Function to install from .deb
+install_deb() {
+    echo "Installing .deb package $PACKAGE_FILE..."
+    dpkg -i "$PACKAGE_FILE"
+}
+
+# Function to install from .rpm
+install_rpm() {
+    echo "Installing .rpm package $PACKAGE_FILE..."
+    rpm -i "$PACKAGE_FILE"
```

Comment:
List `rpm` as a dependency.

Reply:
`rpm` is not really a dependency, but needed for installation of `rpm` package. Added a warning message if it doesn't exist in d9983426075d5ff886fc0c16e40e8000bc078d5a